johnzon-jsonb/pom.xml

```diff
@@ -144,6 +144,7 @@
         <artifactId>maven-bundle-plugin</artifactId>
         <configuration>
           <instructions>
+            <Automatic-Module-Name>johnzon.jsonb</Automatic-Module-Name>
```

Comment:
To drop if we use module-info?

Reply:
yes using a module-info file is a better option if possible.

Reply:
Well module-info or automatic name but all modules should have the same option (it is the key). module-info has some pitfalls and will break some integration - why we dont have it yet - but since we move to 2.0 very soon I guess door can be opened now if preferred but clearly a single solution for the full project :pray: .

Reply:
ok for this PR I'll keep it to simply using the _Automatic-Module-Name_ approach. With the goal of moving over to using a module-info file in the future.

Reply:
done